.github/workflows/community-pr-welcome.yml

```
@@ -0,0 +1,44 @@
+name: Community PR Welcome Message
+
+# This workflow automatically adds a welcome message to PRs from community contributors (forks)
+# It includes a link to the local connector development documentation and other helpful resources
+#
+# MANUAL TESTING INSTRUCTIONS:
+# To manually test this workflow without a real fork PR, temporarily comment-out the fork condition in the
+# `if:` clause and uncomment the "synchronize" event type as a workflow trigger.
+# Then the workflow will run for all new commits.
+#
+# Before merging, remember to again comment-out the "synchronize" clause and uncomment the `if:` condition.
+
+on:
+  pull_request:
+    types:
+      - opened
+      # Toggle this line, uncommenting for testing:
+      - synchronize
+
+jobs:
+  welcome-community-contributor:
+    name: Welcome Community Contributor
+    runs-on: ubuntu-24.04
+    permissions:
+      pull-requests: write
+    # Comment-out this line for manual testing:
+    # if: github.event.pull_request.head.repo.fork == true
```

aside - 

Before merging:

```suggestion
    # Comment-out this line for manual testing:
    if: github.event.pull_request.head.repo.fork == true
```